src/app/services/leaderBoard.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable } from '@angular/core';
+import { environment } from '../../environments/environment';
+import { APIService } from './api.service';
+import 'rxjs/Rx';
+import { map } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { MatDialog, MatDialogConfig } from '@angular/material';
+import { LeaderBoardComponent } from '../leader-board/leader-board.component';
+import { ActivityPoints } from '../leader-board/PointsConfig';
+
+@Injectable()
+
+export class LeaderBoardService {
+
+  private _baseAPIUrl = environment.baseURL + environment.apiEndpoint;
+  private _getLeaderBoard = this._baseAPIUrl + 'leaderBoard';
+  private _getScore = this._baseAPIUrl + 'user/score';
+  private _updateScore = this._baseAPIUrl + 'user/score/update';
+  private readonly dialogMinimumWidth = '700px';
```

Comment:
can be a constant?

Reply:
@Raxerz are you talking about **dialogMinimumWidth** here?

Reply:
Yes. Its a constant right?

Reply:
yep, its read only has to be. Naming convention..

Reply:
Okay! Done.